.gitignore

```diff
@@ -1 +1,8 @@
+# Compiled source #
 build
+main.aux
+main.log
+main.out
+main.pdf
+main.synctex.gz
+main.toc
```

Comment:
FWIW I set my latex system to put built files in a `build/` directory (hence the top line of this file). No problem adding the other lines too, for folks who build in-place.